src/pages/iou/IOUDetailsModal.js

```diff
@@ -150,6 +151,7 @@ class IOUDetailsModal extends Component {
                         ))}
                     </View>
                 )}
+                <OfflineIndicator containerStyles={[styles.ml5, styles.mb2]} />
```

Comment:
Correct me if I'm wrong, but I think this might not be needed now that we added a global offline indicator [with this PR](https://github.com/Expensify/App/pull/9895). Does this not duplicate that indicator?

Reply:
It strangely wasn't covered. `IOUModal` was covered but not the details page

Reply:
Looks like this branch doesn't have those changes yet. I merged main and gave it a test here:

![Screen Shot 2022-08-05 at 3 13 09 PM](https://user-images.githubusercontent.com/6833644/183145521-5ee595ac-80d7-42a9-b789-e810f9e5d279.png)

That global offline indicator will display on mobile devices. On desktop we only display the indicator below the chat message compose box.